sem_covid/entrypoints/notebooks/topic_modeling/plsa/pwdb_topic_modeling_plsa.ipynb

```diff
@@ -0,0 +1,324 @@
+{
```

Comment:
<p>What would be a good way of comparing different models? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/meaningfy-ws/sem-covid/pull/191/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>